### PR TITLE
fixing the AV for graceful shutdown on Win7, set forwardWindowsAuthTo…

### PR DESCRIPTION
**Pull request by pan-wang**
fixing the #53 #57 #58 @jhkimnew @shirhatti

### CODE REVIEW

**Review by shirhatti — APPROVED** [review]

File: src/AspNetCore/Inc/applicationmanager.h [hunk 1]
```diff
@@ -141,7 +141,7 @@ class APPLICATION_MANAGER
              <li> Enable logging the application process' stdout messages </li> \
              <li> Attach a debugger to the application process and inspect </li></ul></fieldset> \
              <fieldset><h4> For more information visit: \
-             <a href=\"http://go.microsoft.com/fwlink/?linkid=808681\" <cite> http://go.microsoft.com/fwlink/?LinkID=808681 </cite></a></h4> \
+             <a href=\"https://go.microsoft.com/fwlink/?linkid=808681\" <cite> https://go.microsoft.com/fwlink/?LinkID=808681 </cite></a></h4> \
```
Comment (moozzyk) [inline src/AspNetCore/Inc/applicationmanager.h]: I don't know this code but the `<a` seems to be missing a closing angle bracket (before `<cite>`)

Comment (shirhatti) [reply src/AspNetCore/Inc/applicationmanager.h]: @moozzyk Thanks for catching that. Fixed in 5641cc5291a21b168f8c8cf42744618baa9ab7bb

**Comment by daboul** [comment]
Do you know when this new 1.0.1972 version including those fixes will be packaged in a runtime? I'm not familiar with this process.  Thanks.

**Comment by jhkimnew** [comment]
@shirhatti Do you know when 1.0.1972 will be available?

**Comment by felipepessoto** [comment]
I also need it

**Comment by simonealessandrelli** [comment]
Me too, I cant wait so I'm going to build aspnetcore module by myself, but of course an official release is the way.

**Comment by shirhatti** [comment]
@simonealessandrelli 1.0.1972 has been available for a while now with the latest version of server hosting bundle. It's available [here](https://www.microsoft.com/net/download/core#/runtime).

**Pull request merged**